src/main/webapp/app/quiz-statistics/quiz-statistics-footer/quiz-statistics-footer.component.html

```diff
@@ -0,0 +1,72 @@
+                        <fa-icon [icon]="['far', 'list-alt']"></fa-icon>
+                        <span class="hidden-xs hidden-sm" jhiTranslate="showStatistic.chooseStatistic"></span>
+                        <span class="caret"></span>
+                    </button>
+                    <div ngbDropdownMenu *ngIf="quizExercise">
+                        <button class="dropdown-item" routerLink="/quiz/{{ quizExercise.id }}/quiz-point-statistic" jhiTranslate="showStatistic.quizPointStatisticTitle"></button>
+                        <button class="dropdown-item" routerLink="/quiz/{{ quizExercise.id }}/quiz-statistic" jhiTranslate="showStatistic.quizStatisticTitle"></button>
+                        <div *ngIf="quizExercise.quizQuestions">
+                            <div *ngFor="let question of quizExercise.quizQuestions; let i = index">
+                                <button
+                                    *ngIf="question.type === MULTIPLE_CHOICE"
+                                    class="dropdown-item"
+                                    routerLink="/quiz/{{ quizExercise.id }}/multiple-choice-question-statistic/{{ question.id }}"
+                                >
+                                    {{ i + 1 }}. {{ question.title | truncate }}
+                                </button>
+                                <button
+                                    *ngIf="question.type === DRAG_AND_DROP"
+                                    class="dropdown-item"
+                                    routerLink="/quiz/{{ quizExercise.id }}/drag-and-drop-question-statistic/{{ question.id }}"
+                                >
+                                    {{ i + 1 }}. {{ question.title | truncate }}
+                                </button>
+                                <button
+                                    *ngIf="question.type === SHORT_ANSWER"
+                                    class="dropdown-item"
+                                    routerLink="/quiz/{{ quizExercise.id }}/short-answer-question-statistic/{{ question.id }}"
+                                >
+                                    {{ i + 1 }}. {{ question.title | truncate }}
+                                </button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div *ngIf="isQuizPointStatistic" id="remaining-time" class="form-group col-sm">
+                <div *ngIf="waitingForQuizStart">
+                    <span jhiTranslate="artemisApp.quizExercise.waitingForStart"></span>
+                </div>
+                <div *ngIf="!waitingForQuizStart">
+                    <span jhiTranslate="artemisApp.quizExercise.remainingTime" class="colon-suffix"></span>
+                    <span
+                        id="remaining-time-value"
+                        [ngClass]="{
```

Comment:
I think it's easier to read and maintain if you do it this way:
```
[ngClass.time-critical]=...
[ngClass.time-warning]=...
```
Also is it on purpose that it can be both critical and warning? E.g. when `remainingTimeSeconds` = 50.